cursorless-talon/src/actions/call.py

```diff
@@ -6,5 +6,5 @@
 
 
 def run_call_action(target: dict):
-    targets = [target, IMPLICIT_TARGET]
+    targets = [target, IMPLICIT_TARGET.copy()]
```

Comment:
should we just make a function that returns a new implicit target every time instead?  This copy stuff is a bit of a nightmare to have to remember

Reply:
Yes do that

Reply:
https://github.com/cursorless-dev/cursorless/issues/719